### ISSUE TITLE
about training loss

### ISSUE THREAD

**Issue by zjz19960805** [vt-vl-lab/iCAN#33]
hi
thanks for your code sharing
and when I try to train a model used VCOCO datasets, I was doubt why the loss is so small from the beginning, like this:
![image](https://user-images.githubusercontent.com/44387104/64781557-bc72e300-d595-11e9-95ab-4c86771fb319.png)
I want to know if I make some mistake to do the training step or the loss is same as yours?
thanks again~
